File: hospitalManagement/InPatient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hospitalManagement
{
   internal class InPatient:Patient
    {
        //Field	
        // Infor field	
        private string room;

        // Management field	
        // Dynamic field	
        // Properties	
        public string Room { get => room; set => room = value; }

        // Constructors	
        public InPatient()
        {

        }

        public InPatient(string firstName
        , string lastName
        , string dateOfBirth
        , string ethnic
        , string religion
        , string nationality
        , string numberOfHealthInsurance
        , string career
        , string address
        , string phonenumber
        , int gentle
        , string relative
        , string phoneNumberOfRelative
        , Fee fees
        , ComeAndLeaveDate admissionDates
        , string room) : base(firstName, lastName, dateOfBirth, ethnic, religion, nationality, numberOfHealthInsurance, career, address, phonenumber, gentle, relative, phoneNumberOfRelative, fees, admissionDates)
        {
            this.FirstName = firstName;
            this.LastName = lastName;
            this.DateOfBirth = dateOfBirth;
            this.Ethnic = ethnic;
            this.Religion = religion;
            this.Nationality = nationality;
            this.NumberOfHealthInsurance = numberOfHealthInsurance;
            this.Career = career;
            this.Address = address;
            this.Phonenumber = phonenumber;
            this.Gentle = gentle;
            this.Relative = relative;
            this.PhoneNumberOfRelative = phoneNumberOfRelative;
            this.Fees = fees;
            this.AdmissionDates = admissionDates;
            this.Room = room;
        }


        // Destructors	
        ~InPatient() { }


        // Methods	

        // in, output

        // General method

        // Other method	

        // Overriding	
        public override void Input()
        {
            base.Input();
            Console.WriteLine("Room: ");
            Room = Console.ReadLine();
        }

        public override void Output()
        {
            base.Output();
            Console.WriteLine($"Room: {Room} ");
        }

        public override string ToString()
        {
            return base.ToString();
        }
    }
}

File: src/hospitalManagement/InPatient.java
package hospitalManagement;
import java.util.Scanner;

public class InPatient extends Patient {
    //Field	
        // Information field	
        private String room;

        // Management field	
        // Dynamic field
        static Scanner input = new Scanner(System.in);
        // Properties	

        public String getRoom() {
            return room;
        }

        public void setRoom(String room) {
            this.room = room;
        }

        // Constructors	
        public InPatient()
        {

        }

        public InPatient(String firstName
        , String lastName
        , String dateOfBirth
        , String ethnic
        , String religion
        , String nationality
        , String numberOfHealthInsurance
        , String career
        , String address
        , String phonenumber
        , int gentle
        , String relative
        , String phoneNumberOfRelative
        , Fee fees
        , ComeAndLeaveDate admissionDates
        , String room) 
        {
            super(firstName, lastName, dateOfBirth, ethnic, religion, nationality, numberOfHealthInsurance, career, address, phonenumber, gentle, relative, phoneNumberOfRelative, fees, admissionDates);
            
            this.room = room;
        }

        // Methods	

        // in, output

        // General method

        // Other method	

        // Overriding	
        public void Input()
        {
            super.Input();
            System.out.println("Room: ");
            room = input.nextLine();
        }

        public void Output()
        {
            super.Output();
            System.out.println("Room: " + room);
        }

        public void ExportBill()
        {
            super.ExportBill();
        }

        public float CalcBill()
        {
            return super.CalcBill();
        }

        public String ToString()
        {
            return super.ToString();
        }
}
